sonar-cxx-plugin/src/test/java/org/sonar/plugins/cxx/coverage/CxxCoverageSensorTest.java

```diff
@@ -132,17 +137,20 @@ public void shouldReportNoCoverageWhenInvalidFilesInvalid() {
   @Test
   public void shouldReportCoverageWhenVisualStudioCase() {
```

Comment:
this is the one that filesystem gets mocked, it also simplifies a bit the windows/linux issues